<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>G-Translate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="srcLanguage"/>
      </item>
      <item>
       <widget class="QTextEdit" name="srcTextEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="translateButton">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>64</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Translate</string>
        </property>
        <property name="text">
         <string>Go Translate&gt;&gt;&gt;</string>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="dstLanguage"/>
      </item>
      <item>
       <widget class="QTextEdit" name="dstTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
